drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -393,6 +490,30 @@ static int sdw_compute_bus_params(struct sdw_bus *bus)
 			__func__, bus->params.bandwidth);
 		return -EINVAL;
 	}
+out:
+	/* multilane can be used */
+	if (m_lane > 0) {
```

Comment:
I think you should initialize m_lane to 0 at declaration.

Reply:
done, thanks.